app/controllers/adminpanel_controller.rb

```diff
@@ -0,0 +1,47 @@
+# Administration Panel
```

Comment:
I think this comment is unnecessary, comment for describe class should be next to class.